kratos/tests/test_model_part_io.py

```diff
@@ -4,13 +4,18 @@
 import KratosMultiphysics
 
 import os
+import sys
 
 
 def GetFilePath(fileName):
     return os.path.dirname(os.path.realpath(__file__)) + "/" + fileName
 
 
 class TestModelPartIO(KratosUnittest.TestCase):
+    
+    def setUp(self):        
+        if (sys.version_info < (3, 2)):
+            self.assertRaisesRegex = self.assertRaisesRegexp

```

Comment:
The problem is that as @maceligueta says it is a different call. What I can do is to hide it behind the KratosUnittest class. Would that help?